packages/flutter_markdown/README.md

```diff
@@ -0,0 +1,12 @@
+# Flutter Markdown
+
+A markdown renderer for Flutter. It supports the [original format](https://daringfireball.net/projects/markdown/), but no inline html.
+
+## Getting Started
+
+Using the Markdown widget is simple, just pass in the source markdown as a string:
+
+    new Block(
```

Comment:
This block should be a ScrollableViewport.  Also, why does the client need to supply the block?  It seems like the Markdown widget could supply the scrollability.